interview-mr/src/test/java/com/sovrn/interview/mr/AverageScoreTest.java

```diff
@@ -0,0 +1,93 @@
+package com.sovrn.interview.mr;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+import com.sovrn.interview.mr.AverageScore;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AverageScoreTest {
+	private final String outputPath = "junit-output";
```

Comment:
Minor quibble, but could have made a separate output path for each test so all output is preserved after test failures, but it's pretty easy to run tests individually if they fail, too. I'm running out of lunch break :)